app/assets/stylesheets/common/_markus.scss

```diff
@@ -1458,3 +1458,21 @@ canvas {
 .jcrop-centered {
   display: inline-block;
 }
+
+// For Actions row
```

Comment:
I reviewed this and overall while I appreciate the desire to make the options look nice, I don't think this change is consistent with the rest of the MarkUs UI. Please instead follow the style of existing tables: use the *tags* table as a model (under an assignment's Settings -> Tags page).